model/perf.go

```diff
@@ -684,12 +762,20 @@ func GetTimeSeriesIds(ctx context.Context, env cedar.Environment) ([]TimeSeriesI
 				},
 			},
 		},
-		{
+	}
+}
+
+func timeSeriesIdPipeline() []bson.M {
+	return append(timeSeriesIdPipelineNoRootReplacement(),
+		bson.M{
 			"$replaceRoot": bson.M{
 				"newRoot": "$_id",
 			},
-		},
-	}
+		})
+}
+
+func GetTimeSeriesIds(ctx context.Context, env cedar.Environment) ([]TimeSeriesId, error) {
+	pipeline := timeSeriesIdPipeline()
```

Comment:
I'd just call the function inline on the next line rather than declare the variables.

Reply:
Think that's refactored out now.